README.md

```diff
@@ -41,9 +41,25 @@ A package to report dependency metrics. Currently relies on output from `yarn in
 - `packageName`: Name of the npm package to retrieve metrics for
 - `version`: Package version to calculate health metrics for
 - `additionalMetricCalculation`: A function which is passed the result of
-  `yarn info`, and whose return value is spread into the return value of this
+  `getPackageVersionInfo`, and whose return value is spread into the return value of this
   function. Allows for customization of metrics without needing to call
-  `yarn info` again.
+  `getPackageVersionInfo` again.
+- `getPackageVersionInfo`: A function which is passed the `packageName` and whose return value is a json string:
```

Comment:
Is it a JSON string or the parsed object thats returned here?

Reply:
Good catch, it's an object, updated